xdsl/parser.py

```diff
@@ -727,21 +728,21 @@ def parse_optional_attribute(self,
 
         # Attribute with default format
         if parse_with_default_format:
-            if not issubclass(attr_def, ParametrizedAttribute):
+            if not attr_def.is_Parametrized:
```

Comment:
I would really prefer using the `issubclass` here, since that helps the IDE to understand what is going on,
and also since this represent exactly what we are doing.

Reply:
will revert